src/core/data-cite.js

```diff
@@ -59,7 +59,14 @@ function requestLookup(conf) {
     switch (elem.localName) {
       case "a": {
         if (elem.textContent === "" && elem.dataset.lt !== "the-empty-string") {
-          elem.textContent = title;
+          // support bikeshed-like section links https://tabatkins.github.io/bikeshed/#section-links
+          // eg: [[[spec/path.html#frag]]] becomes <a href="spec/path.html#frag">[spec/path#frag]</a>
+          if (originalKey.includes("#")) {
+            // ! and ? are for respec to check whether something is normative or not. readers do not need it.
```

Comment:
```suggestion
```

Reply:
Could you explain why you'd rather remove this? 

I added it because it explains the `why` of `why are we supporting [[[?spec/path#frag]]] and [[[!spec/path#frag]]] in addition to [[[spec/path#frag]]] ?`

Reply:
It's kinda _obvious_, we support this everywhere we have a spec citation.

Reply:
@sidvishnoi i'm new to the project and to specs, and it isn't obvious to me. I poked through documentation and the closest I found was

`Normative and informative references work similarly for [[[FOO]]] as they work for [[FOO]], and [[[?FOO]]] can be used to have a non-normative reference in a normative section.`

in the [respec editor's guide](https://github.com/w3c/respec/wiki/ReSpec-Editor's-Guide)

Perhaps the solution is to beef up documentation, since I could not find a reference to `[[!FOO]]` or something similar in the docs. This is something both editors and devs should be aware of.

Reply:
I don't mind these comments so much if they help with the flow of understanding the code. Even though they are obvious, they indeed would help new contributors understand a bit more about what is going on. Having said that, the "!" is sorta/kinda deprecated, because ReSpec does "smart citations" based on context.

Reply:
@marcoscaceres hm. in that case should we account for a deprecated feature in a new feature? leaning towards no if we want it to be really deprecated and eventually gone.

Reply:
Not sure I follow? Just to be clear, we can never drop support for "!" because there are potentially hundreds of specs using it (and a lot of those specs are no longer maintained but important nonetheless), but we actively discourage people from using "!" via spec reviews, etc.